packages/vuepress-theme-odyssey/styles/_Visual.scss

```diff
@@ -0,0 +1,82 @@
+.docskit-visual {
+  grid-column: 2 / 3;
+  min-width: 420px;
+  margin-bottom: $spacing-m;
+
+  .docskit-visual--title {
+    &::before {
+      margin-right: $spacing-xs;
+      font-size: $size-title-5;
+    }
+  }
+
+  .docskit-visual--figure {
+    display: grid;
+    position: relative;
+    grid-auto-columns: max-content;
+    justify-content: center;
+    padding: $spacing-l $spacing-l;
+    border: 1px solid var(--border-color-display, #{$border-color-display});
+    border-radius: $base-border-radius;
+  }
+
+  &.layout-wide {
```

Comment:
I try to avoid variant classes that don't include the component name in them unless they're universal utilities. Not a blocker since we don't have official guidelines yet - just noting.

Reply:
when we do document our conventions, we can do a larger pass